### PR TITLE
[WIP] Flamethrower guitar

### PR DESCRIPTION
**Pull request by fragsnruins**
#### Summary

SUMMARY: Content "Adds a flamethrower electric guitar."

#### Purpose of change

I liked the suggestion by /u/KoboldCommando in the [reddit changelog thread](https://old.reddit.com/r/cataclysmdda/comments/c46ker/cdda_changelog_june_23_2019/erv959n/).

#### Describe the solution

Edits to json files to create the flamethrower_guitar gun and its recipes. As part of this, introduced a craftable ```guitar_amplifier``` item which is an electric guitar with a jury-rigged, tiny amp built in to it. It's much louder than the base electric guitar.

#### Describe alternatives you've considered

Not wanting to ride eternal, shiny and chrome.

#### Additional context

* CR Concerns
  * The intent is for the flamethrower to be played just like ```guitar_amplifier```. I'm able to play for one turn in testing, but it doesn't continuously play like the other musical instruments. Is this just something that can't be neatly implemented into a gun item without c++ edits? Or is it best to change the flamethrower guitar to a tool item and try to embed the gun info? A more seasoned contributor may be able to save me a lot of time with some guidance. Otherwise, I'll try some more attempts, leave as-is (you can kind of play it), or just remove the musical instrument action.

### CODE REVIEW

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -2077,6 +2087,44 @@
     "coverage": 10,
     "material_thickness": 2
   },
+  {
+    "type": "TOOL_ARMOR",
+    "id": "guitar_amplifier",
+    "name": "amplifier electric guitar",
```
Comment (Night-Pryanik) [inline data/json/items/tool_armor.json]:
```suggestion
    "name": "electric guitar amplifier",
```

**Comment by LaVeyanFiend** [comment]
How are you meant to learn the recipe for the guitar amplifier? There's no ```autolearn: true`` or ```book_learn``` on it

**Comment by glenmack** [comment]
Please also add this to the no flaming weapons items blacklist.

File: data/json/items/tool_armor.json (hunk 1 above)
Comment (fragsnruins) [reply data/json/items/tool_armor.json]: Yeah, my naming of that item was too ambiguous. In order to not get the item confused with an actual guitar amplifier, I renamed it to "amplified electric guitar".

File: data/json/items/tool_armor.json [hunk 2]
```diff
@@ -2077,6 +2087,73 @@
...
+    "type": "TOOL_ARMOR",
+    "id": "guitar_amplified",
+    "name": "amplified electric guitar",
+    "copy-from": "guitar_electric",
+    "description": "An electric guitar with a crude, built-in amplifier and speaker.  Now you can melt faces, figuratively speaking.  Unless you do something absurd like strap a flamethrower to it, which would make the melting literal.",
+    "weight": 3000,
+    "use_action": {
+      "type": "musical_instrument",
+      "speed_penalty": 15,
+      "volume": 40,
```
Comment (anothersimulacrum) [inline data/json/items/tool_armor.json]:
```suggestion
      "volume": "10 L",
```

File: data/json/items/tool_armor.json [hunk 3]
```diff
@@ -2077,6 +2087,73 @@
...
+    "flags": [ "BELTED", "TRADER_AVOID" ]
+  },
+  {
+    "id": "flamethrower_guitar",
+    "copy-from": "guitar_amplified",
+    "type": "TOOL_ARMOR",
+    "name": "flamethrower guitar",
+    "description": "A simple, home-made flamethrower strapped onto an electric guitar with a built-in amp.  This impractical device of musical mayhem is likely to lead to a pyrotechnics mishap for someone.",
+    "weight": 3907,
+    "volume": 50,
```
Comment (anothersimulacrum) [inline data/json/items/tool_armor.json]:
```suggestion
    "volume": "12500 ml",
```

**Comment by ZhilkinSerg** [comment]
Please ping me to reopen this PR when you are ready to continue where you have left.

**Pull request closed without merging**